lighthouse-core/audits/time-to-interactive.js

```diff
@@ -17,6 +17,8 @@ const Formatter = require('../formatters/formatter');
 //   https://www.desmos.com/calculator/jlrx14q4w8
 const SCORING_POINT_OF_DIMINISHING_RETURNS = 1700;
 const SCORING_MEDIAN = 5000;
+// This aligns with the external TTI targets in https://goo.gl/yXqxpL
```

Comment:
I've thrown together https://github.com/GoogleChrome/lighthouse/issues/948 but am happy to detail more on there if needed. Is that what you had in mind?